@here/generator-harp.gl/test/test.js

```diff
@@ -17,34 +13,35 @@ const {
 
 */
 describe("harp.gl:app", function () {
-
     this.timeout(300000);
 
     let helperDirectory;
 
     before(async function () {
-        helperDirectory = await yeomanTestHelpers.run(path.join(__dirname, '../generators/app'))
+        helperDirectory = await yeomanTestHelpers
+            .run(path.join(__dirname, "../generators/app"))
             .inTmpDir(function (dir) {
                 if (process.env.USE_NPMRC) {
                     const targetPath = path.join(dir, ".npmrc");
                     fs.copyFileSync(process.env.USE_NPMRC, targetPath);
                 }
             })
             .withPrompts({
-                'access_token': 'test_token'
+                language: "typescript",
```

Comment:
these match what is queried in the Generator, i.e. language & apikey. The `packagename` already has a default, so this isn't required.